daemon/cmd/hubble.go

```diff
@@ -139,8 +139,15 @@ func (d *Daemon) launchHubble() {
 		)
 	}
 
-	if len(option.Config.HubbleRedact) > 0 {
-		parserOpts = append(parserOpts, parserOptions.Redact(logger, option.Config.HubbleRedact))
+	if option.Config.HubbleRedactEnabled {
```

Comment:
Per the comment above this would become:

```suggestion
	if option.Config.HubbleRedactHttpURLQuery || option.Config.HubbleRedactKafkaApiKey {
```

or do you see a case where a user would not set `--hubble-redact-enabled` but not one of the other options?

Reply:
[The idea](https://github.com/cilium/cilium/issues/23887#issuecomment-1632167762.) is that users can probably just set `--hubble-redact-enabled` and then rely on the defaults. Those that want more "granularity" can define the specific http, kafka, etc settings as well.

Would that make sense?

Reply:
Thanks for the context, that makes sense. Sorry I overlooked it in the PR description.